### DISCUSSION TITLE
Add a way to get the Windows 10 accent ccolour as hex code

### DISCUSSION THREAD

**Discussion by cwrenhold** [JanDeDobbeleer/oh-my-posh#2305] [Ideas]
Related to #2302, would it be possible to add a way to get the hex code for the user's current accent colour in OMP? That way we can use the user's currently selected accent colour to generate other colours, for example to colour a series of segments in a row with a gradient based on the user's accent colour.

**Comment by JanDeDobbeleer** [comment]
@cwrenhold https://github.com/JanDeDobbeleer/oh-my-posh/pull/2306

**Reply by cwrenhold** [reply]
Thank you for this, this looks like it'll be amazing!
